obs-websocket-dotnet/Types/OBSScene.cs

```diff
@@ -38,6 +46,7 @@ public OBSScene(JObject data)
         /// <summary>
         /// Default Constructor for deserialization
         /// </summary>
-        public OBSScene() { }
+        public OBSScene() {
```

Comment:
No need to move this to a new line, this is a clear indication of an empty function. Revert to previous syntax